source/main.cpp

```diff
@@ -573,18 +358,20 @@ Conversation LoadConversation()
 #ifdef _WIN32
 void InitConsole()
 {
-	// If both stdout and stderr are already initialized (e.g. writing to a file), do nothing.
 	const int UNINITIALIZED = -2;
 	bool redirectStdout = _fileno(stdout) == UNINITIALIZED;
 	bool redirectStderr = _fileno(stderr) == UNINITIALIZED;
 	bool redirectStdin = _fileno(stdin) == UNINITIALIZED;
-
+	
```

Comment:
Looks like some whitespace got eaten in this method.

Reply:
Normalized